setup.py

```diff
@@ -17,6 +17,8 @@
     numpy
```

Comment:
Can we remove the requirements we don't need? I assume we don't actually need numpy or pysam. , not sure about dataclasses

Reply:
done